src/kube-runtime/src/parse.py

```diff
@@ -36,14 +36,35 @@ def get_container_port(envs, name):
 def export(k, v):
     print "export %s='%s'" % (k, v)
 
+def gen_static_env(framework):
```

Comment:
Static Envs should be generated in Rest-server so metrics can use them later.
This is temp solution.